local-setup/.env

```diff
@@ -0,0 +1,53 @@
+
+## -----------------------------------
+# litentry-worker start options value
+## -----------------------------------
+# The followings are default value.
+# Can be skipped; or overwrite within non-production mode.
+AliceWSPort=9946
+AliceRPCPort=9936
+AlicePort=30336
+BobWSPort=9947
+BobRPCPort=9937
+BobPort=30337
+CollatorWSPort=9944
+CollatorRPCPort=9933
+CollatorPort=30333
+TrustedWorkerPort=2000
+UntrustedWorkerPort=2001
+MuRaPort=3443
+UntrustedHttpPort=4545
+NODE_ENV=local
+
+## -----------------------------------
+# litentry-worker dataproviders config
+## -----------------------------------
+
+# The following key/token are MANDATORY to give when running worker.
+# Otherwise request-vc might suffering from data provider error.
+TWITTER_AUTH_TOKEN_V2=
+DISCORD_AUTH_TOKEN=
+ACHAINABLE_AUTH_KEY=
+ONEBLOCK_NOTION_KEY=
+NODEREAL_API_KEY=
+GENIIDATA_API_KEY=
+
```

Comment:
Might have missed this newly introduced config: https://github.com/litentry/litentry-parachain/blob/92d705fe5ef44d5f53aca31dd380830d78cd9fe8/local-setup/.env.dev#L34

Might worth check others too.

Reply:
egh, this one should be gitignored. Thanks for the raising it 👍

Reply:
.env file shouldn't be here at all, though for .env.dev probably we will add new variables